.github/workflows/test.yaml

```diff
@@ -36,8 +36,18 @@ jobs:
         kind version
         kind create cluster
         kind export kubeconfig
+    - name: Install the rukpak k8s provisioner
+      run: |
+        make generate
+        kubectl create namespace rukpak-system
```

Comment:
Should this get installed by running `k apply -f manifests` and we just create a `namespace.yaml` file in there?

Reply:
I haven't give this a ton of thought yet, but if we introduce a root namespace manifest does that imply all provisioners all share the same system namespace? These steps mainly emulate the run-local Makefile target, so I was making it consistent with how it applied the provisioner.

Reply:
Tangential/Food for thought: do we want to have a distinction between `run-local` and running in CI? Seems like a more productive setup would entail CI jobs that use the same workflows as developers use locally.

Reply:
That brings into question how an end-user gets this onto their cluster and we're probably not there quite yet. Ultimately though we should iron out that UX to make it as simple as possible. Ideally a user wouldn't need to run "k create namespace rukpak-system" though it isn't a huge deal.

Reply:
I think we still need to move all the OLM/registry kind provisioner stuff to an o-f/test-infra and use that as a shared library so we don't have to constantly re-invent the wheel here.